core/arch/arm/include/mm/tee_mm_def.h

```diff
@@ -32,37 +32,4 @@
 #define SMALL_PAGE_MASK		0x00000fff
 #define SMALL_PAGE_SIZE		0x00001000
 
-/*
```

Comment:
I have that in another patch that I intend to upload after this PR, but I can add it here if you prefer that.